src/Features/RulesMissingDocumentation.md

```diff
@@ -0,0 +1,4 @@
+# Rules without documentation
+
+Rule ID | Missing Help Link | Title |
+--------|-------------------|-------|
```

Comment:
All the current IDExxxx rules are documented?

Reply:
No there are undocumented rules. I'm looking now into why it's not failing CI.

Reply:
It would be good to push a dummy commit to this PR that adds a new IDExxxx rule without updating this file and confirm this leads to a CI failure. Then push another dummy commit to add the entry and verify CI passes. You can revert both the commits subsequently.